app/assets/javascripts/Main.js.elm

```diff
@@ -128,93 +133,99 @@ mainUpdate msg mainModel =
                 _ ->
                     ( mainModel, Cmd.none )
 
+        DismissNotice ->
+            ( withoutNotice mainModel, Cmd.none )
+
         _ ->
             case mainModel of
                 InitializingVR route _ settings ->
                     case msg of
                         MapViewportChangedVR mapViewport ->
-                            (InitializedVR mapViewport settings)
+                            (InitializedVR mapViewport settings Nothing)
                                 ! [ Routing.navigate (Routing.routeFromResult route) ]
 
                         _ ->
-                            Debug.crash "map is not initialized yet"
+                            -- Ignore other actions until map is initialized
+                            ( mainModel, Cmd.none )
 
                 Page pagedModel common ->
-                    case pagedModel of
-                        HomeModel homeModel ->
-                            case msg of
-                                HomeMsg msg ->
-                                    updatePagedModel HomeModel common <|
-                                        case msg of
-                                            AppHome.FacilityClicked facilityId ->
-                                                ( homeModel, navigateFacility facilityId )
+                    case ( pagedModel, msg ) of
+                        ( HomeModel homeModel, HomeMsg (AppHome.UnhandledError) ) ->
+                            ( withGenericNotice mainModel, Cmd.none )
 
-                                            AppHome.ServiceClicked serviceId ->
-                                                ( homeModel, navigateSearchService serviceId )
+                        ( HomeModel homeModel, HomeMsg msg ) ->
+                            updatePagedModel HomeModel common <|
+                                case msg of
+                                    AppHome.UnhandledError ->
+                                        -- handled above
+                                        ( homeModel, Cmd.none )
```

Comment:
This should be a Debug.crash since it is unreachable. There is an `Utils.unreachable` but should be called with unit `()` since it is a function.